app/src/main/java/polyrooms/polyrooms/MainActivity.kt

```diff
@@ -17,19 +18,56 @@ import androidx.core.content.ContextCompat
 import kotlinx.android.synthetic.main.activity_main.*
 import java.io.Serializable
 import java.util.*
+import android.R.id.edit
+import android.content.SharedPreferences.Editor
+import com.google.gson.Gson
+import android.text.method.TextKeyListener.clear
+import android.R.id.edit
+import android.animation.AnimatorListenerAdapter
+import android.content.SharedPreferences
+
+
+
 
 class MainActivity : AppCompatActivity() {
 
     var ghour: Int = -1
     var gmin: Int = -1
     var gday: String = "null"
 
+    private var shortAnimationDuration: Int = 0
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         requestWindowFeature(Window.FEATURE_NO_TITLE)
         window.setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN,
                 WindowManager.LayoutParams.FLAG_FULLSCREEN)
         setContentView(R.layout.activity_main)
+
+        val sharedPreferences = getSharedPreferences("production", Context.MODE_PRIVATE)
+
+        //STORE A CUSTOM OBJECT TO SHARED PREFERENCES
```

Comment:
Unused code

Reply:
See above

Reply:
This comment refers to the commented out code